OptimizelySDKCore/OptimizelySDKCore.xcodeproj/project.pbxproj

```diff
@@ -2054,7 +2054,7 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = "OPTLYJSONModelTests/OPTLYJSONModelTests-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
we don't need to test through iOS 8, we have minimum compatibility with iOS 9.

Reply:
I've taken your advice and have changed local_travis.sh to only test >= 9.0 which saves us from cluttering up the OPTLYJSONModelTests with any new code which is sensitive to the iOS version.

@kellyroach-optimizely	Change 8.4 to 9.0 in local_travis.sh	Verified		ba69951